lib/db/mysql/patches/patch-016-017.sql

```diff
@@ -0,0 +1,5 @@
+--  Add `lastUsedAt` column to the `refreshTokens` table.
+
+ALTER TABLE refreshTokens ADD COLUMN lastUsedAt TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP;
```

Comment:
I recently discovered a neat trick where you can ask MySQL to fail out if the `ALTER TABLE` connect be done in an online manner:

  https://github.com/mozilla/fxa-auth-db-mysql/blob/master/lib/db/schema/patch-028-029.sql#L9

I think it's worth adding `ALGORITHM = INPLACE, LOCK = NONE;` to basically all our migrations going forward.  It's no guarantee they'll be performant in production, but it's a good way to catch ones that definitely won't be.